Add caveat for different GUILD_CREATE payload for large guilds (#2587)
hammerandchisel/discord-api-docs: docs/topics/Gateway.md

#### Request Guild Members

> Used to request all members for a guild or a list of guilds. When initially connecting, if you are using [Gateway Intents](#DOCS_TOPICS_GATEWAY/gateway-intents) and don't have the `GUILD_PRESENCES` intent, or if the guild is over 75k members, it will only send members who are in voice, plus the member for you (the connecting user). Otherwise, if a guild has over `large_threshold` members (value in the [Gateway Identify](#DOCS_TOPICS_GATEWAY/identify)), it will only send members who are online, have a role, have a nickname, or are in a voice channel, and if it has under `large_threshold` members, it will send all members. If a client wishes to receive additional members, they need to explicitly request them via this operation. The server will send [Guild Members Chunk](#DOCS_TOPICS_GATEWAY/guild-members-chunk) events in response with up to 1000 members per chunk until all members that match the request have been sent.

Reviewer: > if you are using [Gateway Intents](#DOCS_TOPICS_GATEWAY/gateway-intents) and don't have

Can change to

> if you don't have

Then it's consistent with the warn block. After that, LGTM!

> > warn
> > If you are using [Gateway Intents](#DOCS_TOPICS_GATEWAY/gateway-intents), members and presences returned in this event will only contain your bot and users in voice channels unless you specify the `GUILD_PRESENCES` intent.
> 
> > warn
> > Currently for guilds over 75k members, members and presences returned in this event will only contain your bot and users in voice channels, similar to the Gateway Intents behavior.

Reviewer 2: might be better to merge both into one block?

```suggestion
> warn
> If your bot does not have the `GUILD_PRESENCES` [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents) or the guild has over 75k members, members and presences returned in this event will only contain your bot and users in voice channels.
```

(also updated the intent text to reflect the intent update)

Author: sure, thanks!
